core/build.gradle

```
@@ -5,36 +5,38 @@ version = '0.15-SNAPSHOT'
 archivesBaseName = 'libdohj-core'
 
 dependencies {
-    compile 'org.bitcoinj:bitcoinj-core:0.15.9'
-    compile 'com.madgag.spongycastle:core:1.58.0.0'
+    implementation 'org.bouncycastle:bcprov-jdk15to18:1.68'
+    implementation 'org.bitcoinj:bitcoinj-core:0.15.10'
```

Change to 
```suggestion
    api 'org.bitcoinj:bitcoinj-core:0.15.10'
```

This requires to also change java to java-library on line 1

---

Done